### ISSUE TITLE
ExecutionPoints are not monotonically increasing

### ISSUE THREAD

**Issue by lhchavez** [RecordReplay/node#17]
* **Version**: v16.0.0
* **Platform**: Linux 693f93ffa32a 5.4.0-1043-gcp #46-Ubuntu SMP Mon Apr 19 19:17:04 UTC 2021 x86_64 GNU/Linux
* **Subsystem**:

### What steps will reproduce the bug?

If I set up several breakpoints, sometimes when looking for the next continue targets, the ExecutionPoints are not always monotonically increasing. here's a small trace from a session against https://replay.io/view?id=6dabd641-7891-470e-b2aa-e6c52e06a7f9 :

```
INFO:root:Moving forward from 0 to {'point': '3458764449396031495', 'time': 0.005393452348848498, 'frame': [{'sourceId': '95', 'line': 2, 'column': 2}], 'reason': 'breakpoint'}
INFO:root:Moving forward from 3458764449396031495 to {'point': '4611685954002878504', 'time': 0.008090178523272746, 'frame': [{'sourceId': '95', 'line': 4, 'column': 18}], 'reason': 'breakpoint'}
INFO:root:Moving backwards from 4611685954002878504 to {'point': '31224573109202812141809', 'time': 73.03273825575751, 'frame': [{'sourceId': '95', 'line': 17, 'column': 15}], 'reason': 'breakpoint'}
INFO:root:Moving forward from 31224573109202812141809 to {'point': '42752635233766675054906', 'time': 99.99730327382558, 'frame': [{'sourceId': '95', 'line': 22, 'column': 2}], 'reason': 'breakpoint'}
```

### What do you see instead?

Even though the `time`s are increasing, the ExecutionPoints are not.

### Additional information

Fiddling with the protocol directly, I cannot reproduce this from replay.io

**Comment by jasonLaster** [comment]
Can you share a bit more context on what you were doing? The logs seem right, but i'm not sure what i should be looking for.

**Comment by lhchavez** [comment]
yes, i was trying to interleave the console messages and make them appear in the stdout of the process as the execution goes forwards in time. the above log was obtained by setting a couple of breakpoints and calling [`Debugger.findResumeTarget`](https://replay.io/protocol/tot/Debugger#method-findResumeTarget) a few times. the non-monotonicity of the ExecutionPoints means that I can't know whether an ExecutionPoint happened before or after each of the console messages D:
